src/modules/samples/actions.js

```diff
@@ -3,8 +3,11 @@ import api from "../../utils/api";
 import {DEFAULT_PAGINATION_LIMIT} from "../../config";
 
 export const GET           = createNetworkActionTypes("SAMPLES.GET");
+export const SET_FILTER = "SAMPLES.SET_FILTER";
+export const CLEAR_FILTERS = "SAMPLES.CLEAR_FILTERS";
```

Comment:
Some lines are `=`-aligned some are not. They should either all be aligned or none.